frontend/src/components/donation/donorForest/DonorForestEntry.js

```diff
@@ -27,21 +27,20 @@ const useStyles = makeStyles((theme) => ({
 
 const getWidthFromStep = (step) => {
   const widths = {
-    "1": "65px",
```

Comment:
`yarn format` got rid of these quotation marks. Just want to make sure they were not necessary in this case.

Reply:
I think you test that you'd have to add a donation by a user and check if the /donorforest page works as expected